documentation/docs/get-started/demo-explorer-get-started/installation.md

```diff
@@ -47,3 +48,16 @@ cd packages/demo-explorer && pnpm dev
 ```
 
 Now you can see the VCKit Demo Explorer at [`http://localhost:3000`](http://localhost:3000)
+
+## Common cases
+
+:::warning
+**IMPORTANCE**: Every time you make changes to the agent configuration, you need to remove your agent data on Explorer by go to the `manage` on the top right corner and click `Remove Agent` on your agent, it will automatically remove and load the new agent configuration.
```

Comment:
I'd like to update the statement like this "Every time you make changes to the agent configuration, you need to remove your agent data on Explorer by hovering on the `Agent` profile in the top right corner, then selecting `manage` and clicking `Remove Agent` on your agent, it will automatically remove and load the new agent configuration." this is clearer to understand